apps/omg/lib/omg/state.ex

```diff
@@ -284,7 +277,7 @@ defmodule OMG.State do
   end
 
   defp do_form_block(state) do
-    {:ok, child_block_interval} = Eth.RootChain.get_child_block_interval()
```

Comment:
this was a call to configuration:
```
def get_child_block_interval(), do: Application.fetch_env(:omg_eth, :child_block_interval)
```
so now instead of reading this from config, we read it once on process init and it stays in the process state